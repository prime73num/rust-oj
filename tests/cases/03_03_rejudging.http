###
# Send request:
PUT http://127.0.0.1:12345/jobs/0 HTTP/1.1
content-type: application/json

{}
###
# Got response:
# HTTP 404
# content-length: 35
# content-type: application/json
# date: Sat, 03 Sep 2022 07:22:46 GMT
# 
# {"code":3,"reason":"ERR_NOT_FOUND"}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() {let mut line1 = String::new();std::io::stdin().read_line(&mut line1).unwrap();let a: i32 = line1.trim().parse().unwrap();let mut line2 = String::new();std::io::stdin().read_line(&mut line2).unwrap();let b: i32 = line2.trim().parse().unwrap();println!(\"{}\", a + b);}","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 1186
# content-type: application/json
# date: Sat, 03 Sep 2022 07:22:46 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":0},{"id":1,"info":"","memory":0,"result":"Accepted","time":0},{"id":2,"info":"","memory":0,"result":"Accepted","time":0},{"id":3,"info":"","memory":0,"result":"Accepted","time":0},{"id":4,"info":"","memory":0,"result":"Accepted","time":0},{"id":5,"info":"","memory":0,"result":"Accepted","time":0},{"id":6,"info":"","memory":0,"result":"Accepted","time":0},{"id":7,"info":"","memory":0,"result":"Accepted","time":0},{"id":8,"info":"","memory":0,"result":"Accepted","time":0},{"id":9,"info":"","memory":0,"result":"Accepted","time":0},{"id":10,"info":"","memory":0,"result":"Accepted","time":0}],"created_time":"2022-09-03T07:22:46.887Z","id":0,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() {let mut line1 = String::new();std::io::stdin().read_line(&mut line1).unwrap();let a: i32 = line1.trim().parse().unwrap();let mut line2 = String::new();std::io::stdin().read_line(&mut line2).unwrap();let b: i32 = line2.trim().parse().unwrap();println!(\"{}\", a + b);}","user_id":0},"updated_time":"2022-09-03T07:22:46.887Z"}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"C","problem_id":0,"source_code":"#include <stdio.h>\nint main() { printf(\"Hello World!\\n\"); return 0; }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 446
# content-type: application/json
# date: Sat, 03 Sep 2022 07:22:47 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":0},{"id":1,"info":"","memory":0,"result":"Accepted","time":0}],"created_time":"2022-09-03T07:22:47.280Z","id":1,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"C","problem_id":0,"source_code":"#include <stdio.h>\nint main() { printf(\"Hello World!\\n\"); return 0; }","user_id":0},"updated_time":"2022-09-03T07:22:47.280Z"}
###
# Send request:
PUT http://127.0.0.1:12345/jobs/0 HTTP/1.1
content-type: application/json

{}
###
# Got response:
# HTTP 200
# content-length: 1186
# content-type: application/json
# date: Sat, 03 Sep 2022 07:22:47 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":0},{"id":1,"info":"","memory":0,"result":"Accepted","time":0},{"id":2,"info":"","memory":0,"result":"Accepted","time":0},{"id":3,"info":"","memory":0,"result":"Accepted","time":0},{"id":4,"info":"","memory":0,"result":"Accepted","time":0},{"id":5,"info":"","memory":0,"result":"Accepted","time":0},{"id":6,"info":"","memory":0,"result":"Accepted","time":0},{"id":7,"info":"","memory":0,"result":"Accepted","time":0},{"id":8,"info":"","memory":0,"result":"Accepted","time":0},{"id":9,"info":"","memory":0,"result":"Accepted","time":0},{"id":10,"info":"","memory":0,"result":"Accepted","time":0}],"created_time":"2022-09-03T07:22:47.329Z","id":0,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() {let mut line1 = String::new();std::io::stdin().read_line(&mut line1).unwrap();let a: i32 = line1.trim().parse().unwrap();let mut line2 = String::new();std::io::stdin().read_line(&mut line2).unwrap();let b: i32 = line2.trim().parse().unwrap();println!(\"{}\", a + b);}","user_id":0},"updated_time":"2022-09-03T07:22:47.329Z"}
###
# Send request:
PUT http://127.0.0.1:12345/jobs/1 HTTP/1.1
content-type: application/json

{}
###
# Got response:
# HTTP 200
# content-length: 446
# content-type: application/json
# date: Sat, 03 Sep 2022 07:22:47 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":0},{"id":1,"info":"","memory":0,"result":"Accepted","time":0}],"created_time":"2022-09-03T07:22:47.721Z","id":1,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"C","problem_id":0,"source_code":"#include <stdio.h>\nint main() { printf(\"Hello World!\\n\"); return 0; }","user_id":0},"updated_time":"2022-09-03T07:22:47.721Z"}
###
# Send request:
PUT http://127.0.0.1:12345/jobs/2 HTTP/1.1
content-type: application/json

{}
###
# Got response:
# HTTP 404
# content-length: 35
# content-type: application/json
# date: Sat, 03 Sep 2022 07:22:47 GMT
# 
# {"code":3,"reason":"ERR_NOT_FOUND"}
