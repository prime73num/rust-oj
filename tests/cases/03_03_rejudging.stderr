[2022-09-03T07:22:45Z INFO  actix_server::builder] Starting 2 workers
[2022-09-03T07:22:45Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime
[2022-09-03T07:22:46Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET / HTTP/1.1" 404 0 "-" "-" 0.000317
[2022-09-03T07:22:46Z INFO  actix_web::middleware::logger] 127.0.0.1 "PUT /jobs/0 HTTP/1.1" 404 35 "-" "-" 0.000325
[2022-09-03T07:22:46Z INFO  post_jobs_handler] Post jobs
[2022-09-03T07:22:47Z INFO  post_jobs_handler] job run success
[2022-09-03T07:22:47Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 1186 "-" "-" 0.383748
[2022-09-03T07:22:47Z INFO  post_jobs_handler] Post jobs
[2022-09-03T07:22:47Z INFO  post_jobs_handler] job run success
[2022-09-03T07:22:47Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 446 "-" "-" 0.044909
[2022-09-03T07:22:47Z INFO  actix_web::middleware::logger] 127.0.0.1 "PUT /jobs/0 HTTP/1.1" 200 1186 "-" "-" 0.388023
[2022-09-03T07:22:47Z INFO  actix_web::middleware::logger] 127.0.0.1 "PUT /jobs/1 HTTP/1.1" 200 446 "-" "-" 0.044267
[2022-09-03T07:22:47Z INFO  actix_web::middleware::logger] 127.0.0.1 "PUT /jobs/2 HTTP/1.1" 404 35 "-" "-" 0.000352
